src/integration/common/SEO.js

```diff
@@ -171,14 +171,16 @@ export default () => {
       expect(metaTagContent).toMatchSnapshot();
     });
 
-    it('Linked data', () => {
-      const linkedDataEl = document.querySelector(
-        'script[type="application/ld+json"]',
-      );
-
-      expect(linkedDataEl).toBeInTheDocument();
-      expect(linkedDataEl.textContent).toBeTruthy();
-      expect(JSON.parse(linkedDataEl.textContent)).toMatchSnapshot();
-    });
+    const linkedDataEl = document.querySelector(
+      'script[type="application/ld+json"]',
+    );
+
+    if (linkedDataEl) {
```

Comment:
is this a canonical only test?

Reply:
Just checked - we get linked data on both AMP & Canonical pages. It's just failing at the moment for on demand episode pages, because we have not done the work. Should I check for page type instead?

Reply:
yeh because otherwise that test would pass if linked data was removed by mistake

Reply:
`pageType` is on the global scope

Reply:
However, the page type (which is parsed from the test file path) is un-camelCased https://github.com/bbc/simorgh/blob/latest/src/integration/integrationTestEnvironment.js#L15

maybe we should not do this.

Reply:
I think it is safer to leave this as a broken test until the metadata work is done - should be completed within the next few days. I'll remove the if check, and just keep this branch up to date with latest until it's resolved.